cmd/killgrave/main.go

```diff
@@ -1,13 +1,16 @@
 package main
 
 import (
-	"log"
+	"os"
+
+	log "github.com/sirupsen/logrus"
 
 	"github.com/friendsofgo/killgrave/internal/app"
 )
 
 func main() {
 	if err := app.Run(); err != nil {
-		log.Fatal(err)
+		log.Error(err)
+		os.Exit(1)
```

Comment:
If we're going to use `logrus`, which I'm not sure yet, but looks promising, then I think we can just keep the `log.Fatal(err)`, as it also calls `os.Exit(1)`, and has its own mechanism for running "exit handlers", if needed.

Reply:
lol, i didn't notice, replacing it